### ISSUE TITLE
[REQ] Component Request: RSSI

### ISSUE THREAD

**Issue by brentru** [adafruit/Wippersnapper_Components#126]
Sends the RSSI value from a WiFi development board back to Adafruit IO at a configurable interval.

**Comment by brentru** [comment]
@ladyada   Are you interested in having @tyeth  look into this? I'm suuuper interested in this as ESPHome/HA does it, and it doesn't require an extra sensor

**Comment by ladyada** [comment]
yeah this is interesting! however it would require a change to the backend right?

**Comment by ladyada** [comment]
ok add it to the pile :)

**Comment by brentru** [comment]
@ladyada  would this be something we'd want to add in api-v2?

**Comment by ladyada** [comment]
yeah its a good idea!!

**Comment by tyeth** [comment]
This has been surfaced in the main Wippersnapper firmware, as part of beta-88, but no RSSI component / backend support yet.

**Comment by brentru** [comment]
@ladyada  After Tyeth finishes the v2 display feature, I think this is a fun and smaller task for him to take on after a long-running/large task.

**Comment by ladyada** [comment]
yes!!
